test/no-ripple-test.js

```diff
@@ -339,39 +264,37 @@ suite('Default ripple', function() {
   });
 
   test('Set default ripple on account, check new trustline', function(done) {
-    var steps = [
-      function (callback) {
-        testutils.create_accounts(
-          $.remote,
-          'root',
-          '10000.0',
-          [ 'alice', 'bob' ],
+    async.series([
+      function(callback) {
+        $.create_accounts(
+          root,
+          [ alice, bob ],
           { default_rippling: false },
           callback);
       },
-      function (callback) {
-        var tx = $.remote.createTransaction('AccountSet', {
-          account: 'bob',
+      function(callback) {
+        let tx = $.remote.createTransaction('AccountSet', {
+          account: bob,
           set_flag: 8
         });
-        testutils.submit_transaction(tx, callback);
+        $.submit_transaction(tx, callback);
       },
-      function (callback) {
-        var tx = $.remote.createTransaction('TrustSet', {
-          account: 'root',
-          limit: '100/USD/alice'
+      function(callback) {
+        let tx = $.remote.createTransaction('TrustSet', {
+          account: root,
+          limit: `100/USD/${alice}`
```

Comment:
They are an es2015 feature. They can go across multiple lines, and support expression interpolation, via `${ }` 

The newer versions of ripple-lib gutted the facility to configure account aliases so now rather than lookup an address from the issuer segment 'alice' , the string passed in is already in form:  `100/USD/rPzy....`.